src/main/java/webserver/ResponseBody.java

```diff
@@ -23,27 +23,25 @@ public static String getHtml(String path) throws IOException {
   }
 
   private static Path createFilePath(String path) {
-    Path filePath = FileSystems.getDefault().getPath(path.substring(1));
-    return filePath;
+      return FileSystems.getDefault().getPath(path.substring(1));
   }
 
   private static FileInputStream getFile(Path path) {
       try {
-          FileInputStream file = new FileInputStream(path.toString());
-          return file;
+          return new FileInputStream(path.toString());
       } catch (FileNotFoundException e) {
           Path errorFile = createFilePath("/404.html");
           return getFile(errorFile);
       }
   }
 
   private static String createHTMLString(FileInputStream file) throws IOException {
-      StringBuffer buf = new StringBuffer();
+      StringBuilder buf = new StringBuilder();
```

Comment:
Agreed; you might say what it contains rather than what it is.  i.e. call it `html`

that's based on my pet peeve of saying things like `StringBuilder builder = new StringBuilder` -- where you say the word "builder" 3 times.